sds_in_a_box/sds_in_a_box_stack.py

```diff
@@ -102,3 +102,26 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
                                                       )
                                         )
         indexer_lambda.apply_removal_policy(cdk.RemovalPolicy.DESTROY)
+
+        # download query API lambda
+        download_query_api = lambda_.Function(self,
+                                          id="DownloadQueryAPILambda",
+                                          function_name='download-query-api',
+                                          code=lambda_.Code.from_asset(os.path.join(os.path.dirname(os.path.realpath(__file__)), "SDSCode/")),
+                                          handler="download_query_api.lambda_handler",
+                                          role=lambda_role,
+                                          runtime=lambda_.Runtime.PYTHON_3_9,
+                                          timeout=cdk.Duration.seconds(60),
+                                          environment={
+                                              "URL_EXPIRE": "86400"
+                                          }
```

Comment:
We could get rid of this with my above comment if you defaulted to 86400 over there. I'm fine either way though, just something to consider.